<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceWidget</class>
 <widget class="QWidget" name="DeviceWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>364</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <item>
    <widget class="QFrame" name="border">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(125, 128, 128)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>-1</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QFrame" name="verticalFrame">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,1,5,1,2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="WidgetNameLabel">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>19</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: white</string>
             </property>
             <property name="text">
              <string>InputName</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
           <item>
            <widget class="QLabel" name="PresetNameLabel">
             <property name="styleSheet">
              <string notr="true">color: white</string>
             </property>
             <property name="text">
              <string>PresetName</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="RemovePresetButton">
             <property name="styleSheet">
              <string notr="true">color: white;</string>
             </property>
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: white</string>
             </property>
             <property name="text">
              <string>MIDI From</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="DevicesBox">
             <property name="styleSheet">
              <string notr="true">color: white</string>
             </property>
             <property name="placeholderText">
              <string>Devices</string>
             </property>
             <item>
              <property name="text">
               <string>No Device</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Computer Keyboard</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <property name="styleSheet">
              <string notr="true">color: white</string>
             </property>
             <property name="placeholderText">
              <string>Channels</string>
             </property>
             <item>
              <property name="text">
               <string>All Channels</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Ch 1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Ch 2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Ch 3</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true">color: white</string>
             </property>
             <property name="text">
              <string>Volume</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="VolumeSlider">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>70</number>
             </property>
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="MuteButton">
             <property name="styleSheet">
              <string notr="true">color: white;</string>
             </property>
             <property name="text">
              <string>Mute</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="EnableButton">
             <property name="styleSheet">
              <string notr="true">color: white;</string>
             </property>
             <property name="text">
              <string>Enable</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="styleSheet">
              <string notr="true">color: white;</string>
             </property>
             <property name="text">
              <string>Note OFF disabled</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
